doc/images.md

```diff
@@ -45,6 +45,18 @@ Sprites can also be used as widgets, to do so, just use `Flame.util.spriteAsWidg
 
 A complete example of using sprite as widgets can be found [here](examples/animation_widget).
 
+## SpriteBatch
+
+If you have an image atlas ( an image with smaller images inside ), and would like to render it effectively - `SpriteBatch` does the job.
+
+Give it the filename of the image, and then add rectangles which describes various part of the image, in addition to transforms ( position, scale and rotation ) and optional colors.
+
+You render it with an `Canvas` and an optional `Paint`, `BlendMode` and `CullRect`.
+
+An `SpriteBatchComponent` is also available for your convenience.
+
+See example [here](examples/sprite_batch).
```

Comment:
Does this link work? I think it should be `docs/example/sprite_batch` doesn't it?

Reply:
I think you are right. I looked at `examples/animation_widget` which is also wrong then? Will update that also.